pallets/interest-accrual/src/lib.rs

```diff
@@ -422,11 +485,27 @@ impl<T: Config> InterestAccrual<T::InterestRate, T::Balance, Adjustment<T::Balan
 		Pallet::<T>::do_renormalize_debt(old_interest_rate, new_interest_rate, normalized_debt)
 	}
 
-	fn reference_rate(interest_rate_per_sec: T::InterestRate) {
+	fn reference_yearly_rate(
+		interest_rate_per_year: T::InterestRate,
+	) -> Result<T::InterestRate, DispatchError> {
+		Pallet::<T>::reference_yearly_interest_rate(interest_rate_per_year)
+	}
+
+	fn reference_rate(interest_rate_per_sec: T::InterestRate) -> Result<(), DispatchError> {
 		Pallet::<T>::reference_interest_rate(interest_rate_per_sec)
 	}
 
 	fn unreference_rate(interest_rate_per_sec: T::InterestRate) -> Result<(), DispatchError> {
 		Pallet::<T>::unreference_interest_rate(interest_rate_per_sec)
 	}
+
+	fn verify_penalty_rate(
```

Comment:
Could we name it to something less connected to write-offs. Maybe just `verify_rate`

Reply:
Really, `verify` is the wrong verb, as it both verifies and converts to a per-second rate. The thing that makes it penalty-specific is that penalty rates are still of the form "0.xxxxxxx" instead of "1.xxxxx" for normal rates. Maybe we want to use a more generic term for "rates which can be used to manipulate other rates" instead of "rates that are meant to stand on their own as *real rates*? I'm not sure